packages/teleterm/src/ui/services/quickInput/quickInputService.ts

```diff
@@ -17,56 +17,55 @@ limitations under the License.
 import { Store, useStore } from 'shared/libs/stores';
 import { CommandLauncher } from 'teleterm/ui/commandLauncher';
 import { ClustersService } from 'teleterm/ui/services/clusters';
+import { WorkspacesService } from 'teleterm/ui/services/workspacesService';
 import * as pickers from './quickPickers';
-import { QuickInputPicker } from './types';
+import { AutocompleteResult, AutocompleteToken, Suggestion } from './types';
 
 type State = {
-  picker: QuickInputPicker;
   inputValue: string;
   visible: boolean;
 };
 
 export class QuickInputService extends Store<State> {
-  quickCmdPicker: pickers.QuickCmdPicker;
-  quickLoginPicker: pickers.QuickLoginPicker;
-  quickDbPicker: pickers.QuickDbPicker;
-  quickServerPicker: pickers.QuickServerPicker;
+  quickCommandPicker: pickers.QuickCommandPicker;
```

Comment:
`private` (same for line 31)

Reply:
`lastFocused` on line 31 is used by `useQuickInput`, though I suspect we'll get rid of this property soon anyway as it's related to the `goBack` behavior which doesn't exist for this implementation of the command bar.